locksmith/migrations/2023101305000000-public-key-unique.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const table = 'UserReferences'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.changeColumn(table, 'publicKey', {
+      type: Sequelize.STRING,
+      unique: true,
+    })
+    await queryInterface.removeConstraint(
+      table,
+      'UserReferences_publicKey_fkey'
+    )
+    await queryInterface.addIndex(table, { fields: ['publicKey'] })
```

Comment:
have you tested this with a forked database to make sure this doesn't cause any issue?

Reply:
I have not, but not sure how to do so I will ping you on Discord :)